examples/seismic/model.py

```diff
@@ -133,7 +133,8 @@ def _initialize_bcs(self, bcs="damp"):
         # First initialization
         init = self.damp is None
         # Get current Function if alread yinitialized
```

Comment:
typo but can be done in other PR

Reply:
oki

Reply:
fixed now since I had to push again anyway